### PR TITLE
Add constructor template in LargestSeriesProduct.php

### PR DESCRIPTION
**Pull request by realwahl**
The tests imply that the Series class has a constructor with a string parameter. If we provide a class template, we should provide a template for the constructor as well.

### CODE REVIEW

**Comment by mk-mxp** [comment]
@realwahl Are you willing to make this mergable?

**Comment by homersimpsons** [comment]
@realwahl Tests are failing because a `[tab]` is used on line 30 instead of 4 spaces. Do you have time to fix this ?

File: exercises/practice/largest-series-product/LargestSeriesProduct.php [hunk 1]
```diff
@@ -26,6 +26,11 @@
 
 class Series
 {
+    public function __construct(string $input)
+	{
```
Comment (homersimpsons) [inline exercises/practice/largest-series-product/LargestSeriesProduct.php:30]: Replace tab by 4 spaces:
```suggestion
    {
```

**Review by homersimpsons — CHANGES_REQUESTED** [review]

**Comment by realwahl** [comment]
I will check, thanks!

**Comment by realwahl** [comment]
@mk-mxp  I think it should be ok now, can you please check?
@homersimpsons Thanks a lot!

**Review by homersimpsons — APPROVED** [review]

**Pull request merged**